### ISSUE TITLE
Nb-datepicker with inputed date

### ISSUE THREAD

**Issue by andredatsch** [akveo/nebular#1081]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [X] bug report
* [ ] feature request

### Issue description

How can i set the nb-datepicker to work with the inputed date?

![157521773bf14f2fddc960c117a5c00f](https://user-images.githubusercontent.com/3539904/49956306-04bc8200-feec-11e8-8844-261e2c63ad33.gif)

**Comment by Tibing** [comment]
Hi @andredatsch, I've tried to reproduce this issue with [showcase in Nebular docs](https://akveo.github.io/nebular/example/datepicker/datepicker-showcase.component) and it works fine for me. Does it work for you? If not, could you please provide a reproducible example with the issue?

**Comment by sameerpallav** [comment]
This always selects today's date if I click on the form. Expected behavior should be it should open the input date by default.

**Comment by yggg** [comment]
Looks like it works now. Checked with native, date-fns and moment date services.
>Have you tryied with the Formatter

Yep. Please notice, you can use format only in pair with date-fns or moment, not native date (It doesn't support formatting).

Does anyone still have issues with writing date into input manually?


@sameerpallav regarding
>This always selects today's date if I click on the form. Expected behavior should be it should open the input date by default.

You can set `date` input of `nb-datepicker` to null. Then it'll just show current month and date and nothing will be selected. Same if you use `formControl` or `ngModel` binding on input. Set value to null, like this:
```
  dateFormControl = new FormControl(null);
  // or
  dateModel = null;
```

**Comment by sameerpallav** [comment]
@yggg my use case was different. For example, if the input field has a value Jan 9 1991, when i click on it it opens today's date by default and not Jan of 1991 and i have to navigate to Jan 1991 manually and 9th is selected. Shldn't it start with Jan 1991 by default if its the input

**Comment by yggg** [comment]
@sameerpallav Now I see it, thanks for clarification 🙂
Fixed in `3.4.1` ([pull request](https://github.com/akveo/nebular/pull/1300)).
